index.html

```diff
@@ -226,7 +226,7 @@ <h3>Without JS</h3>
 
     <!-- WTC Signin JS for rendering that button -->
     <script
-      src="https://cdn-ca.grnspace.co/wtc_signin.min.js"
+      src="https://cdn-ca.grnspace.ca/wtc_signin.min.js"
```

Comment:
@aemitos @evanwchurchill @matthewshirley I dont think we actualy updated this in cloudfront 🤔 .
I get nothing when I go to https://cdn-ca.grnspace.ca/wtc_signin.min.js but I still get something at https://cdn-ca.grnspace.co/wtc_signin.min.js

You can also see none of the demo site works atm either.
https://grnspace.github.io/wtc-oidc-client/

Reply:
This was just merged this morning during standup